code/model/modelread.cpp

```diff
@@ -3211,7 +3211,7 @@ int model_load(const  char* filename, int n_subsystems, model_subsystem* subsyst
 
 	TRACE_SCOPE(tracing::LoadModelFile);
 
-	mprintf(( "Loading model '%s' into slot '%i'\n", filename, num ));
+	nprintf(("Model" "Loading model '%s' into slot '%i'\n", filename, num ));
```

Comment:
This `nprintf` is missing a comma, which effectively means it's missing a category

Reply:
I'm shocked that compiles honestly